### PR TITLE
Add article "WinUI 3 (Windows App SDK) sample app"

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article for review:
* **WinUI 3 (Windows App SDK) sample app**
   * https://review.learn.microsoft.com/microsoft-edge/webview2/samples/webview2-winui3-sample?branch=pr-en-us-3302
   * [Planned live](https://learn.microsoft.com/microsoft-edge/webview2/samples/webview2-winui3-sample)
   * New article.
      * Un-commented planned TOC node.
   * Removed redirect.
   * Un-commented planned row of Samples table.

Related PR: 
* [PR 258](https://github.com/MicrosoftEdge/WebView2Samples/pull/258) (WinUI 3 sample: Update Readme and fixed-version Runtime code) in `WebView2Samples` repo

AB#54887339
AB#54887352 - Samples repo PR

### CODE REVIEW

File: microsoft-edge/webview2/samples/webview2-winui3-sample.md [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+      * `TryNavigate()`
+      * `Go_OnClick(sender, RoutedEventArgs)`
+      * `AddressBar_KeyDown(sender, KeyRoutedEventArgs)`
+      * `SetTitle()`
+      * `GetWebView2Version()`
+
+
+<!-- ---------- -->
+###### Condensed listing
+<!-- todo: trim listing -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/samples/webview2-winui3-sample.md:400]: i wish to condense this code @captainbrosset

File: microsoft-edge/webview2/samples/webview2-winui3-sample.md [hunk 2]
```diff
@@ -0,0 +1,603 @@
...
+    /// </summary>
+    public partial class App : Application
+    {
+        /// <summary>
+        /// Initializes the singleton application object.  Equivalent to main() or WinMain().
+        /// </summary>
+        public App()
+        {
+            this.InitializeComponent();
+            // If you're shipping a fixed-version WebView2 Runtime with your app,
```
Comment (mikehoffms) [inline microsoft-edge/webview2/samples/webview2-winui3-sample.md:314]: i plan to create PR in WebView2Samples repo to update the spelling/ wording/ formatting of this entire comment block
@peiche-jessica

Comment (mikehoffms) [reply microsoft-edge/webview2/samples/webview2-winui3-sample.md:314]: use better version # than 95.x?

File: microsoft-edge/webview2/samples/webview2-winui3-sample.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/samples/webview2-winui3-sample.md:400]: doc-design idea:
* insert h6 heading between each method, creating a spot to add commentary

Comment (mikehoffms) [reply microsoft-edge/webview2/samples/webview2-winui3-sample.md]: closing.  i put a subhead between each method

**Review by captainbrosset — APPROVED** [review]
I can't do a technical check on this as I don't have the right expertise. But the content looks good to me.

File: microsoft-edge/webview2/samples/webview2-winui3-sample.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/samples/webview2-winui3-sample.md]: Here's a link to [PR 258](https://github.com/MicrosoftEdge/WebView2Samples/pull/258) (WinUI 3 sample: Update Readme and fixed-version Runtime code) in WebView2Samples repo.

**Review by peiche-jessica — APPROVED** [review]
LGTM

**Pull request merged**